src/sentry/api/endpoints/assistant.py

```diff
@@ -18,18 +20,28 @@
 
 
 class AssistantSerializer(serializers.Serializer):
-    guide_id = serializers.IntegerField(required=True)
-    status = serializers.ChoiceField(choices=zip(VALID_STATUSES, VALID_STATUSES), required=True)
-    useful = serializers.BooleanField()
+    guide = serializers.CharField(required=False)
+    guide_id = serializers.IntegerField(required=False)
+    status = serializers.ChoiceField(choices=zip(VALID_STATUSES, VALID_STATUSES))
+    useful = serializers.BooleanField(required=False)
+
+    def validate(self, attrs):
+        attrs = super(AssistantSerializer, self).validate(attrs)
+        guide = attrs.get("guide")
+        guide_id = attrs.get("guide_id")
+
+        if guide_id:
```

Comment:
Is it worth still checking that this is a valid id? Maybe not important since this is temporary.